opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -341,6 +341,24 @@ const investigationGraphStixSightingRelationshipQuery = graphql`
     }
 `;
 
+// To count the number of relationships for MetaObjects and Identities.
+// /!\ It counts only rels that point towards given ids. So the value may
+// not be exactly the total number of rels in some cases.
```

Comment:
Can you write the specific cases to be explicit on that point ?
Thank you

Reply:
Done